frontend/src/components/MeetingRoom/MobileHeader/MobileHeader.spec.tsx

```diff
@@ -0,0 +1,69 @@
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import { vi, describe, it, Mock, expect, beforeEach, afterEach } from 'vitest';
+import MobileHeader from './MobileHeader';
+import useSessionContext from '../../../hooks/useSessionContext';
+import useRoomName from '../../../hooks/useRoomName';
+import useRoomShareUrl from '../../../hooks/useRoomShareUrl';
+
+vi.mock('../../../hooks/useSessionContext');
+vi.mock('../../../hooks/useRoomName');
+vi.mock('../../../hooks/useRoomShareUrl');
+
+describe('MobileHeader component', () => {
+  const mockedRoomName = 'test-room-name';
+  let originalClipboard: Clipboard;
+
+  beforeEach(() => {
+    (useRoomName as Mock).mockReturnValue(mockedRoomName);
+    (useRoomShareUrl as Mock).mockReturnValue('https://example.com/room123');
```

Comment:
Nit, we should keep it consistent consistent and use `https://example.com/test-room-name`

Reply:
Can we actually change this to `https://example.com/room/test-room-name` ?
Just to match our actual routes

Reply:
similar to the Tooltip / Fade, github is not showing the latest here. Please see: https://github.com/Vonage/vonage-video-react-app/pull/52/files#diff-adf120fcb4626de5700c62e68974923c5212dbc19556374580bff2e30f1a0eedR29-R33

Reply:
ah okay I'll add it. this thread still shows outdated code, though 😢